src/features/Endpoint/Endpoint.tsx

```diff
@@ -36,6 +36,10 @@ const EndPoint = () => {
     window.location.reload();
   };
 
+  const current_url = `wss://${localStorage.getItem(
+    'config.server_url',
+  )}/websockets/v3?app_id=${localStorage.getItem('config.app_id')}&l=EN&brand=deriv`;
+
```

Comment:
You can put the localstorage calls in seperate variables and then call the variables in the `current_url` one.

So something like this:

```
const server_url = localStorage.getItem('config.server_url');
const app_id = localStorage.getItem('config.app_id');

current_url = `wss://${server_url}/websockets/v3?app_id=${app_id}&l=EN&brand=deriv`;
```

It helps with readability.

Other than that everything looks good to me.

Reply:
Okay